test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -49,7 +49,7 @@ class ApiUsers::PermissionsControllerTest < ActionController::TestCase
     end
 
     should "include a hidden field for the signin permission so that it is not removed" do
-      application = create(:application, with_non_delegatable_supported_permissions: ["perm-1", SupportedPermission::SIGNIN_NAME])
+      application = create(:application, with_non_delegatable_supported_permissions: %w[perm-1])
```

Comment:
Typo in the commit message: `Signon` -> `signin` 🙃

Also, "A factory `application` is always created with a Signon [sic] permission" -> I think it's more that the factory doesn't override the default non-factory behaviour (`after_create` hook). It's not something specific to factory-made applications. You kinda say this at the end though!

Reply:
One day I'll get the difference between Signon and signin correct, but today obviously wasn't that day.